Description: Modification to allow dpkg-buildflags
 This modification allows the dpkg-buildflags to inject compilation
 flags in addition to vendor specific patches. Also this patch removes
 -Os from upstream allowing use DEB_BUILD_OPTIONS=noopt
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: not-needed
Last-Update: 2022-01-16
--- a/config.mk 2022-01-07 12:42:18.000000000 +0100
+++ b/config.mk 2022-01-16 20:33:02.005772998 +0100
@@ -25,10 +25,10 @@ INCS = -I${X11INC} -I${FREETYPEINC}
 LIBS = -L${X11LIB} -lX11 ${XINERAMALIBS} ${FREETYPELIBS}

 # flags
-CPPFLAGS = -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"${VERSION}\" ${XINERAMAFLAGS}
+CPPFLAGS += -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"${VERSION}\" ${XINERAMAFLAGS}
 #CFLAGS   = -g -std=c99 -pedantic -Wall -O0 ${INCS} ${CPPFLAGS}
-CFLAGS   = -std=c99 -pedantic -Wall -Wno-deprecated-declarations -Os ${INCS} ${CPPFLAGS}
-LDFLAGS  = ${LIBS}
+CFLAGS   += -std=c99 -pedantic -Wall -Wno-deprecated-declarations -Os ${INCS} ${CPPFLAGS}
+LDFLAGS  += ${LIBS}

 # Solaris
 #CFLAGS = -fast ${INCS} -DVERSION=\"${VERSION}\"
